.mypy.ini

```diff
@@ -0,0 +1,20 @@
+[mypy]
+
+[mypy-numpy.*]
+# can be removed if numpy 1.20+ (typing annotations available from this version on)
+ignore_missing_imports = True
+
+[mypy-pandas.*]
+ignore_missing_imports = True
+
+[mypy-matplotlib.*]
+ignore_missing_imports = True
+
+[mypy-sklearn.*]
+ignore_missing_imports = True
+
+[mypy-seaborn.*]
+ignore_missing_imports = True
+
+[mypy-scipy.*]
+ignore_missing_imports = True
```

Comment:
Bellissimo! But does this mean that we can skip the " # type: ignore" everywhere in the code (v. cgnal-core)?